tailwind.config.js

```diff
@@ -58,5 +58,5 @@ module.exports = {
         },
     },
     variants: {},
-    plugins: [require("@tailwindcss/ui")],
+    plugins: [require("@tailwindcss/ui", "@tailwindcss/aspect-ratio")],
```

Comment:
We should make a note to remove `aspect-ratio` once it's got more browser support and  is rolled into Tailwind proper.

Reply:
I just left a comment noting that.